spinnaker/spinnaker_system/kube_smoke_test.py

```diff
@@ -230,7 +230,7 @@ def create_server_group(self):
             }],
             'stack': bindings['TEST_STACK'],
```

Comment:
You might want to test both ways now that we can create a service.
(actually it was fixed when I did the original checkin but forgot to add it)